doc/contributing/security-release-process.md

```diff
@@ -60,11 +60,6 @@ The current security stewards are documented in the main Node.js
     * List vulnerabilities in order of descending severity
     * Use the "summary" feature in HackerOne to sync post-release content
       and CVE requests. Example [2038134](https://hackerone.com/bugs?subject=nodejs\&report_id=2038134)
-    * Ask the HackerOne reporter if they would like to be credited on the
```

Comment:
I think this step should stay in, but we could update to ask if they want to be credited, and if so include a link to their H1 profile. I assume the automation should be able to get the original reporter of an issue?

Reply:
From the title it might imply we would already credit them, but I still think we need to ask if they want to be credited, and by removing lines 64-67 it seems to imply there won't be a credit at all which conflicts with the title.

Reply:
So the idea is to always credit them (their H1 handle at least). I don't see why they wouldn't like to be credited as this information will be public anyway when the report gets disclosed.